extensions/quick-latex/src/quick-latex.tsx

```diff
@@ -0,0 +1,56 @@
+import { Action, ActionPanel, environment, List, popToRoot, showHUD } from "@raycast/api";
+import { useState, useEffect } from "react";
+import { existsSync, mkdirSync } from "fs";
+import { resolve } from "path";
+import { runAppleScript } from "run-applescript";
+import { image } from "image-downloader";
+
+const downloadDir = resolve(environment.supportPath, "download");
+const latexUrl = "https://latex.codecogs.com/png.image?" + encodeURIComponent("\\dpi{512}");
+const latexUrlDark = "https://latex.codecogs.com/png.image?" + encodeURIComponent("\\dpi{512}\\bg{white}");
```

Comment:
```suggestion
const latexUrlDark = "https://latex.codecogs.com/png.image?" + encodeURIComponent("\\dpi{512}\\bg{black}\\fg{white}");
```
Maybe change it a bit like this